h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmLoss.java

```diff
@@ -143,6 +167,10 @@
     @Override public double impute(double u) {
       return u > 0? 1 : 0;
     }
+
+    @Override public int mimpute(double[] u) {
+      return ArrayUtils.maxIndex(u);
+    }
```

Comment:
`mimpute` should be defined in conjunction with `mloss` and `mlgrad`, for categorical loss functions only